src/jitlibs/softfloat/CMakeLists.txt

```diff
@@ -1,14 +1,405 @@
+	${EXTERNAL_INSTALL_LOCATION}/source/f128_to_i64_r_minMag.c
+	${EXTERNAL_INSTALL_LOCATION}/source/f128_to_f16.c
+	${EXTERNAL_INSTALL_LOCATION}/source/f128_to_f32.c
+	${EXTERNAL_INSTALL_LOCATION}/source/f128_to_extF80.c
+	${EXTERNAL_INSTALL_LOCATION}/source/f128_to_f64.c
+	${EXTERNAL_INSTALL_LOCATION}/source/f128_roundToInt.c
+	${EXTERNAL_INSTALL_LOCATION}/source/f128_add.c
+	${EXTERNAL_INSTALL_LOCATION}/source/f128_sub.c
+	${EXTERNAL_INSTALL_LOCATION}/source/f128_mul.c
+	${EXTERNAL_INSTALL_LOCATION}/source/f128_mulAdd.c
+	${EXTERNAL_INSTALL_LOCATION}/source/f128_div.c
+	${EXTERNAL_INSTALL_LOCATION}/source/f128_rem.c
+	${EXTERNAL_INSTALL_LOCATION}/source/f128_sqrt.c
+	${EXTERNAL_INSTALL_LOCATION}/source/f128_eq.c
+	${EXTERNAL_INSTALL_LOCATION}/source/f128_le.c
+	${EXTERNAL_INSTALL_LOCATION}/source/f128_lt.c
+	${EXTERNAL_INSTALL_LOCATION}/source/f128_eq_signaling.c
+	${EXTERNAL_INSTALL_LOCATION}/source/f128_le_quiet.c
+	${EXTERNAL_INSTALL_LOCATION}/source/f128_lt_quiet.c
+	${EXTERNAL_INSTALL_LOCATION}/source/f128_isSignalingNaN.c
+	${EXTERNAL_INSTALL_LOCATION}/source/f128M_to_ui32.c
+	${EXTERNAL_INSTALL_LOCATION}/source/f128M_to_ui64.c
+	${EXTERNAL_INSTALL_LOCATION}/source/f128M_to_i32.c
+	${EXTERNAL_INSTALL_LOCATION}/source/f128M_to_i64.c
+	${EXTERNAL_INSTALL_LOCATION}/source/f128M_to_ui32_r_minMag.c
+	${EXTERNAL_INSTALL_LOCATION}/source/f128M_to_ui64_r_minMag.c
+	${EXTERNAL_INSTALL_LOCATION}/source/f128M_to_i32_r_minMag.c
+	${EXTERNAL_INSTALL_LOCATION}/source/f128M_to_i64_r_minMag.c
+	${EXTERNAL_INSTALL_LOCATION}/source/f128M_to_f16.c
+	${EXTERNAL_INSTALL_LOCATION}/source/f128M_to_f32.c
+	${EXTERNAL_INSTALL_LOCATION}/source/f128M_to_extF80M.c
+	${EXTERNAL_INSTALL_LOCATION}/source/f128M_to_f64.c
+	${EXTERNAL_INSTALL_LOCATION}/source/f128M_roundToInt.c
+	${EXTERNAL_INSTALL_LOCATION}/source/f128M_add.c
+	${EXTERNAL_INSTALL_LOCATION}/source/f128M_sub.c
+	${EXTERNAL_INSTALL_LOCATION}/source/f128M_mul.c
+	${EXTERNAL_INSTALL_LOCATION}/source/f128M_mulAdd.c
+	${EXTERNAL_INSTALL_LOCATION}/source/f128M_div.c
+	${EXTERNAL_INSTALL_LOCATION}/source/f128M_rem.c
+	${EXTERNAL_INSTALL_LOCATION}/source/f128M_sqrt.c
+	${EXTERNAL_INSTALL_LOCATION}/source/f128M_eq.c
+	${EXTERNAL_INSTALL_LOCATION}/source/f128M_le.c
+	${EXTERNAL_INSTALL_LOCATION}/source/f128M_lt.c
+	${EXTERNAL_INSTALL_LOCATION}/source/f128M_eq_signaling.c
+	${EXTERNAL_INSTALL_LOCATION}/source/f128M_le_quiet.c
+	${EXTERNAL_INSTALL_LOCATION}/source/f128M_lt_quiet.c
+)
+
+SET(OTHER_HEADERS 
+	${EXTERNAL_INSTALL_LOCATION}/${INC}/opts-GCC.h
+	${EXTERNAL_INSTALL_LOCATION}/build/${PLATFORM_N}/platform.h
+	${EXTERNAL_INSTALL_LOCATION}/${INC}/primitiveTypes.h
+	${EXTERNAL_INSTALL_LOCATION}/${INC}/primitives.h
+	${EXTERNAL_INSTALL_LOCATION}/${INC}/softfloat_types.h
+	${EXTERNAL_INSTALL_LOCATION}/${INC}/internals.h
+	${EXTERNAL_INSTALL_LOCATION}/source/${SPECIALIZE}/specialize.h
+	${EXTERNAL_INSTALL_LOCATION}/${INC}/softfloat.h
+)
+
+SET(CMAKE_C_ARCHIVE_CREATE "<CMAKE_AR> crs <TARGET> <LINK_FLAGS> <OBJECTS>")
```

Comment:
why was this required? a shallow search gave me the impression that this is the default already: https://stackoverflow.com/questions/5659225/how-do-i-set-the-options-for-cmake-ar

Reply:
Good catch. I first modeled the build configuration after the original which used these specific archive commands. Not sure, whether these actually have any effect. On my setup, CMake defaults to `ar qc ...` not `ar crs ...` invoked by the original berkeley softfloat.

Removed with [commit a76b228](https://github.com/JoGei/etiss/commit/a76b22870118113702e31b61e755a697f2f4ca99)